internal/server/models/model.go

```diff
@@ -13,15 +13,20 @@ type Modelable interface {
 	IsAModel() // there's nothing specific about this function except that all Model structs will have it.
 }
 
+const CreatedBySystem = 0
+
 type Model struct {
-	ID uid.ID
-	// CreatedAt is set by GORM to time.Now when a record is first created.
+	ID        uid.ID
+	DeletedAt gorm.DeletedAt
+
+	// Set by GORM to time.Now() when a record is created.
 	// See https://gorm.io/docs/conventions.html#Timestamp-Tracking
 	CreatedAt time.Time
-	// CreatedAt is set by GORM to time.Now when a record is updated.
+	// Set by GORM to time.Now() when a record is updated.
 	// See https://gorm.io/docs/conventions.html#Timestamp-Tracking
 	UpdatedAt time.Time
```

Comment:
What's the reason for this godoc change? I don't think this matches the recommended style.

Godoc comments are supposed to start with the field name (like they were before).

Reply:
Wasn't aware of the recommended style which seems repetitive. Will revert

Reply:
Looking for some docs or examples to make sure I'm not mistaken. Some examples here: https://pkg.go.dev/net/http#Client, https://pkg.go.dev/net/http#PushOptions

I couldn't find it explicitly documented in effective Go for struct fields.